ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx

```diff
@@ -16,12 +16,14 @@ import * as DOMPurify from 'dompurify';
 import { marked } from 'marked';
 import { Box, Theme } from '@mui/material';
 import { PanelProps } from '@perses-dev/plugin-system';
+import { PersesChartsTheme, useChartsTheme } from '@perses-dev/components';
 import { MarkdownPanelOptions } from './markdown-panel-model';
 
 export type MarkdownPanelProps = PanelProps<MarkdownPanelOptions>;
 
-function createMarkdownPanelStyles(theme: Theme) {
+function createMarkdownPanelStyles(theme: Theme, chartsTheme: PersesChartsTheme) {
   return {
+    padding: `${chartsTheme.container.padding.default}px`,
```

Comment:
I had this thought that I might remove the CSS rule below that says to remove the margin on the first element in the panel (see comment below)

The reason for this rule was that most HTML elements have built-in margins (h1, h2, p, etc) and their margins were combining themselves with the panel padding and creating too much space

I was thinking that once we had the ability to adjust padding, we could NOT set the vertical padding, and allow elements to use their built-in margins

However, now I'm thinking that I'm not sure all the HTML elements have built-in margins (tables? links?), so it's probably safer to have padding, and continue to remove the first element's top margin

All of which to say, will you delete that TODO comment that I have below, and maybe just say "Ignore the top margin on the first element because it creates too much space when combined with panel padding" 

(Unless what I'm saying makes sense to you, and you want to take the approach)

Reply:
I agree that it's safer to keep the `:first-child` top padding removal for exactly the reason you mentioned about inconsistent padding on different elements. I'll take out the TODO.